contracts/xasset/src/collateralized.rs

```diff
@@ -110,7 +113,7 @@ pub trait IsCollateralized {
     fn close_cdp(&mut self, lender: Address) -> Result<(), Error>;
 
     /// Updates and returns the accrued interest on a cdp
-    fn get_accrued_interest(&self, lender: Address) -> Result<Interest, Error>;
+    fn get_accrued_interest(&self, lender: Address) -> Result<InterestDetail, Error>;
 
     /// Pay the interest on a CDP
```

Comment:
Commenting here because GH does not allow me to make suggestions for the doc comment for `repay_debt`. We need to update the doc comment for `repay_debt` as discussed on our call yesterday, so that CLI users and other app users will understand the workflow:

1. Call `get_acrued_interest`, record the `approval_amount`
2. Call the XLM SAC's `approve` function with that `approval_amount`
3. Call `repay_debt` within five minutes

While I'm here commenting on the `pay_interest` doc, though, I'd like to ask: what is the difference between the two? Do we need both `repay_debt` and `pay_interest`? If we do, then can we add more detail to this doc too, to make its purpose more clear?